src/hooks/postMessage.ts

```diff
@@ -98,6 +159,10 @@ const configurePostMessageHooks = (queryConfig: QueryClientConfig) => {
           reject(`the type '${type}' for payload '${JSON.stringify(payload)}' is not recognized`);
         }
       } catch (e) {
+        queryConfig.notifier({
+          type: 'Resolution/rejection',
```

Comment:
Question: is it mobile specific?

Reply:
No, it was just missing, and I felt it would be good to send that info if it fails ?